Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM python:3 as app
 ENV PYTHONUNBUFFERED 1
 RUN pip install --upgrade pipenv gunicorn
+RUN pip install pymysql
 WORKDIR /app
 COPY Pipfile /app/
 COPY Pipfile.lock /app/
 RUN pipenv install --deploy --system
+RUN pipenv install mysqlclient
```

Comment:
@sstratoti `pymysql` replaces this requirement, so we don't need to run this here.